### ISSUE TITLE
[ToolbarBadgeSample] The SVG icons do not render on Android

### ISSUE THREAD

**Issue by Emasoft** [kyurkchyan/xamarin-samples#1]
In the ToolbarBadgeSample the SVG icons are correctly displayed on iOS, but they do not render on Android.

**Comment by kyurkchyan** [comment]
Yes. Actually this example is not complete due to exactly that issue. After days of debugging I decided to dig the Xamarin.Forms code. And issue was inside how Xamarin.Forms.Android handle ImageSource for action bar. ImageSource can be Uri, File and so on. On iOS all of these cases are handled. On Android the image is searched only inside bundle. I probably need to fire a bug report to Xamarin.

**Comment by Emasoft** [comment]
Yes please, file a bug report: https://bugzilla.xamarin.com/describecomponents.cgi?product=Forms

**Comment by kyurkchyan** [comment]
I've created a sample project to file a bug report then I found that somebody else already reported. I've added a comment with more details. [HERE](https://bugzilla.xamarin.com/show_bug.cgi?id=43569#c3) it is.

**Comment by kyurkchyan** [comment]
[Here](https://forums.xamarin.com/discussion/77099/toolbaritem-icon-with-badges-could-be-possible-and-with-custom-icon-too/p1?new=1) is a thread for this issue. Add a comment with +1 to promote it.

**Comment by Emasoft** [comment]
Done, thanks!
While we wait I'm currently trying to solve the other problem. The one about the colored rendering.
On the toolbar the image is being interpreted as a template, and as a result only the alpha component is being used. 
To force the image to use the original colors on iOS you need to do something like this in the native renderer:

`image = image.ImageWithRenderingMode(UIImageRenderingMode.AlwaysOriginal);`

Once you do that the image is supposedly rendered as colored.
Unfortunately it doesn't seem to work. But maybe I'm doing something wrong...

**Comment by kyurkchyan** [comment]
That's actually another issue with iOS. I've also posted this inside my comment in [Xamarin Bug Report](https://bugzilla.xamarin.com/show_bug.cgi?id=43569#c3)
For ios there should be a way to make the iOS not tint the icon. This is usually specified in UIImage
 UIImage *image = [UIImage imageNamed:@"myImage.png"];
 image = [image imageWithRenderingMode:UIImageRenderingModeAlwaysOriginal];
So, I believe there should be a special property for this in ToolbarItem. There is no other way of doing this
